docs/sources/dashboards/create-reports/_index.md

```diff
@@ -94,62 +64,71 @@ refs:
     - pattern: /docs/grafana-cloud/
       destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#temp_data_lifetime
-  rendering-configuration:
+  templates-and-variables:
     - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/create-reports/report-settings/#rendering-configuration
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/variables/
     - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/create-reports/report-settings/#rendering-configuration
-  reporting-configuration:
+      destination: /docs/grafana-cloud/visualizations/dashboards/variables/
+  time-range-controls:
     - pattern: /docs/grafana/
-      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/enterprise-configuration/#reporting
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/use-dashboards/#set-dashboard-time-range
     - pattern: /docs/grafana-cloud/
-      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/enterprise-configuration/#reporting
-  
+      destination: /docs/grafana-cloud/visualizations/dashboards/use-dashboards/#set-dashboard-time-range
 ---
 
 # Create and manage reports
 
-**Reporting** allows you to automatically generate PDFs from any of your dashboards and have Grafana email them to interested parties on a schedule. This is available in Grafana Cloud and in Grafana Enterprise.
-
-{{< admonition type="note" >}}
-If you have [Role-based access control](ref:role-based-access-control) enabled, for some actions you would need to have relevant permissions.
-Refer to specific guides to understand what permissions are required.
-{{< /admonition >}}
-
-Any changes you make to a dashboard used in a report are reflected the next time the report is sent. For example, if you change the time range in the dashboard, then the time range in the report also changes, unless you've configured a custom time range.
+**Reporting** allows you to send automated and scheduled emails from any of your dashboards.
+You can configure several elements of these reports and generate PDFs, CSV files, and embedded images.
+Any changes you make to a dashboard used in a report are reflected the next time the report is sent.
 
 ## Requirements
 
-- SMTP must be configured for reports to be sent. Refer to [SMTP](ref:smtp) in [Configuration](ref:configuration) for more information.
-- The Image Renderer plugin must be installed or the remote rendering service must be set up. Refer to [Image rendering](ref:image-rendering) for more information.
+- SMTP must be configured for reports to be sent. Refer to [SMTP configuration documentation](ref:smtp) for more information.
+- The [Grafana image renderer plugin](/grafana/plugins/grafana-image-renderer) (v3.0+) must be installed or the remote rendering service must be set up. Refer to [Image rendering](ref:image-rendering) for more information.
+
```

Comment:
Do these requirements for Enterprise only? Can we clarify what the requirements are for Enterprise vs Cloud?

Reply:
Okay I'm going to add that in just to make it super clear.